apps/yapms/src/lib/components/modals/ModalBase.svelte

```diff
@@ -14,18 +13,19 @@
 		} else if ($store !== undefined) {
 			$store.open = false;
 		}
-		console.log(open);
 	}
 
 	let content: HTMLDivElement | undefined;
 	let offsetHeight: number;
+	let dialog: HTMLDialogElement;
```

Comment:
This just needs to be of type undefined, as it's undefined at initialization. The offsetHeight above it probably also needs to be of type undefined.